python/tvm/script/tir/scope_handler.py

```diff
@@ -639,6 +639,8 @@ def create_loop_info(
         begin, end = [convert(_) for _ in [begin, end]]
         assert self.context and self.node, "call 'exit_scope' before 'enter_scope'"
         extent = end if begin == 0 else self.context.analyzer.simplify(end - begin)
+        if begin == 0 and isinstance(extent, PrimExpr):
+            begin = IntImm(extent.dtype, 0)
```

Comment:
shall we consider providing `span` for `begin`

Reply:
sure, I have updated it